flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/catalog/CatalogTestBase.java

```diff
@@ -177,20 +177,8 @@ protected Map<String, String> getStreamingTableProperties() {
         return new HashMap<String, String>() {
             {
                 put(IS_STREAMING, "true");
-                putAll(getGenericFlag(isGeneric()));
+                put(FactoryUtil.CONNECTOR.key(), "hive");
```

Comment:
I wonder why we have introduced the keyword hive into a non-hive specific module. Can we move the hive specific reference into a hive module as part of this change? I assume setting this as hive is misleading and not useful for a non-hive use case.